### DISCUSSION TITLE
Joining array in Volt

### DISCUSSION THREAD

**Discussion by wurst-hans** [phalcon/cphalcon#15620] [Q&A]
Trying to join an array with newline, i.e. `var|join("\n")` does not work, because Volt translates double quotes to single quotes. Looking into compiled Volt file this becomes `join('\n')`. What can I do?

**Comment by quasipickle** [comment]
You could try the `nl2br` filter:
```php
var|join("<br />")|nl2br
```
Which is a hacky workaround, but it should work.  Alternatively, you could go down to raw PHP
```php
implode("\n",$var)
```
Finally, you could write your own filter.

None of these solutions preclude the possibility that there might just be a way to use `join` somehow.

**Reply by wurst-hans** [reply]
`nl2br` is nothing I want to use in this case, because it should *not* contain any `<br>` tags, as I'm re-using the content in a textarea on client side. I prepare content on service side in controller and pass it to Volt then.
Thanks nevertheless.
